### ISSUE TITLE
Make 0.7.2 release

### ISSUE THREAD

**Issue by Ainali** [publiccodenet/standard#1001]
As we need new printed standards for FOSDEM and State of OpenCon in early February, and we have [a few updates,](https://github.com/publiccodenet/standard/compare/main...develop) we should aim to make a release in mid-January. That should allow us enough time to order new printed versions. 

It would be good if this release included #999.

**Comment by ericherman** [comment]
If by that time we have merged #999, it may be that we want to call it 0.8.0 because even though no codebase would go from passing to failing, there may be some quite substantial changes to the text and tone of many requirements.

**Comment by Ainali** [comment]
Yes, with 801b2715a4be9ea3a0b5ca67844cfb077cf03d19 I agree that a bump to 0.8.0 is called for.

**Comment by Ainali** [comment]
We are aiming to get this released by 10 January.
